### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by menyagah**
# In this milestone:1:
 - I set up a new project with webpack.
 - I created an index.html file and wrote my HTML markup there. 
 - I created an empty To-Do List placeholder.
 - The index.html file is being set as a template using the HTML Webpack Plugin.
 - I created an index.js file and set an array of some simple to-do tasks (array of objects). 
 - I wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
 - On page load, I rendered the dynamically created list of tasks in the dedicated placeholder. 
 - I created a style.css and set rules for the To-Do List. CSS is being loaded by Webpack Style/CSS Loader.
 - My list is a clone of the part of the minimalist project captured in the video given in the requirements.
 - Below is the live link to the project:
 https://raw.githack.com/menyagah/ToDoList-App/feature/dist/index.html
- Fixed TSE errors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "redesigned-webpack",
+  "version": "1.0.0",
+  "description": "A webpack template for future use",
+  "main": "index.js",
+  "scripts": {
```
Comment (rloterh) [inline package.json:6]: - [ ] The _script option_ of your _package.json_ file is **not properly configured** and hence your application cannot start using the command `npm start`. Kindly add the necessary _script options_ similar to as can be seen below:

**Set package.json script options:**
image


**Incomplete script configuration:**
image

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "url": "git+https://github.com/menyagah/redesigned-webpack.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/menyagah/redesigned-webpack/issues"
+  },
+  "homepage": "https://github.com/menyagah/redesigned-webpack#readme",
+  "devDependencies": {
```
Comment (rloterh) [inline package.json:21]: - [ ] The _dervServer_ package is missing your application. Kindly install the same and add that in your **webpack** configuration to enable your application to be run on the server locally.

**devServer in webpack configuration file:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Martin, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by menyagah** [comment]
Thank you @rloterh

**Review by AshakaE — APPROVED** [review]
### Hi @menyagah  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making requested changes
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by menyagah** [comment]
Thank you @AshakaE !

**Pull request merged**